Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "wayland-kbd"
 version = "0.2.0"
 documentation = "http://vberger.github.io/wayland-kbd/wayland_kdb/"
```

Comment:
I suppose I could just modify this `patch-1` branch, but that sounds like a lot of work :)